http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/monitor/BaseCloudServiceRuntimeMonitor.scala

```diff
@@ -193,7 +193,7 @@ abstract class BaseCloudServiceRuntimeMonitor[F[_]] {
         )
         _ <- clusterQuery.detachPersistentDisk(runtimeAndRuntimeConfig.runtime.id, ctx.now).transaction
         _ <- curStatus match {
-          case RuntimeStatus.Deleted =>
+          case RuntimeStatus.Deleted | RuntimeStatus.Deleting =>
```

Comment:
my worry with this is that then a runtime will be stuck in deleting, moving the runtime to error allows a user to then delete the runtime

Reply:
that's a good point...maybe we can think about making the error msg a bit better